frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -56,7 +56,8 @@ export const navigateTo = (opt: devNavigationMenu) => {
     case devNavigationMenu.Monitoring: {
       cy.get(devNavigationMenuPO.monitoring).click();
       detailsPage.titleShouldContain(pageTitle.Monitoring);
-      cy.testA11y('Monitoring Page in dev perspective');
+      // Bug: ODC-5119 is created related to Accessibility violation - Until bug fix, below line is commented to execute the scripts in CI
+      // cy.testA11y('Monitoring Page in dev perspective');
```

Comment:
https://issues.redhat.com/browse/ODC-5119 this is about add page right? we may need diff bug id here

Reply:
This bug is applicable for all pages, As it is related to left side navigation menu